<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckRegister</class>
 <widget class="QDialog" name="CheckRegister">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>634</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Check Register</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="deleteChecksButtons">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>De&amp;lete</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>430</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Mark Selected Checks</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
   <widget class="QPushButton" name="buttonClearSel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Cleared</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonVoidSel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Void</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOutSel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Outstanding</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3Table" name="tabSummary">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>420</width>
     <height>210</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="numRows">
    <number>3</number>
   </property>
   <property name="numCols">
    <number>3</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <row>
    <property name="text">
     <string>Cleared</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Outstanding</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Total</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Deposits</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Checks</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Balance</string>
    </property>
   </column>
  </widget>
  <widget class="Q3ButtonGroup" name="filterGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>440</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>Show</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading</set>
   </property>
   <widget class="QRadioButton" name="dateOption">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Dated:</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Q3DateEdit" name="registerFromDate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="showOutBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Outstanding</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="showVoidBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Void</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="showClearBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cleared</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Q3DateEdit" name="registerToDate">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_2_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="toNumberEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fromNumberEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="numberOption">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Numbered:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonReport">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>&amp;Report</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNewDep">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>New &amp;Deposit</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="postChecksButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Po&amp;st</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttChkNo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>&amp;Next Chk #</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNewChk">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>New &amp;Check</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDone">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabChecks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>870</width>
     <height>340</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_chks">
    <attribute name="title">
     <string>Checks</string>
    </attribute>
    <widget class="QLabel" name="textLabel1_2_3">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Index By:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="indexCombo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Date</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Number</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Payee</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amount</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Status</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="searchEdit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel1">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter Payee:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="payeeFilterEdit">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="Q3Table" name="checkTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>850</width>
       <height>240</height>
      </rect>
     </property>
     <property name="dragAutoScroll">
      <bool>false</bool>
     </property>
     <property name="numRows">
      <number>0</number>
     </property>
     <property name="numCols">
      <number>5</number>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>Q3Table::MultiRow</enum>
     </property>
     <property name="focusStyle">
      <enum>Q3Table::FollowStyle</enum>
     </property>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payee</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>S</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_dep">
    <attribute name="title">
     <string>Deposits</string>
    </attribute>
    <widget class="Q3ListView" name="listViewDeposits">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>770</width>
       <height>290</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>Q3ListView::Multi</enum>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>S</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Number</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Memo</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3DateEdit</class>
   <extends>QWidget</extends>
   <header>Qt3Support/Q3DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabChecks</tabstop>
  <tabstop>buttonClearSel</tabstop>
  <tabstop>buttonVoidSel</tabstop>
  <tabstop>buttonOutSel</tabstop>
  <tabstop>showClearBox</tabstop>
  <tabstop>showVoidBox</tabstop>
  <tabstop>showOutBox</tabstop>
  <tabstop>dateOption</tabstop>
  <tabstop>registerFromDate</tabstop>
  <tabstop>registerToDate</tabstop>
  <tabstop>numberOption</tabstop>
  <tabstop>fromNumberEdit</tabstop>
  <tabstop>toNumberEdit</tabstop>
  <tabstop>deleteChecksButtons</tabstop>
  <tabstop>postChecksButton</tabstop>
  <tabstop>buttChkNo</tabstop>
  <tabstop>buttonNewChk</tabstop>
  <tabstop>buttonNewDep</tabstop>
  <tabstop>buttonReport</tabstop>
  <tabstop>buttonDone</tabstop>
  <tabstop>tabSummary</tabstop>
  <tabstop>listViewDeposits</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/checkdao.h</include>
  <include location="global">vector</include>
  <include location="local">dao/setupdao.h</include>
  <include location="local">dao/salemanager.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
